db/product_seeds.csv

```diff
@@ -0,0 +1,26 @@
+name,description,price,photo_url,stock,available
```

Comment:
One question, which param are the numbers after true for each line? Is it user_id? 
```suggestion
name,description,price,photo_url,stock,available, user_id
```